code/modules/antagonists/slaughter/imp_antag.dm

```diff
@@ -18,3 +19,10 @@
 	newobjective.explanation_text = "Try to get a promotion to a higher devilish rank."
 	newobjective.owner = owner
 	objectives += newobjective
+
+/datum/antagonist/slaughter/ui_static_data(mob/user)
```

Comment:
this isnt a slaughter demon

Reply:
good point

Reply:
were imps not removed when we removed devil stuff?